Detectors/gconfig/g3Config.C

```diff
@@ -27,7 +27,7 @@ void Config()
              << endl;
   }
   // create Fair Specific Stack
-  o2::Data::Stack *st = new AliceO2::Data::Stack();
+  o2::Data::Stack *st = new o2::Data::Stack();
   st->SetMinPoints(0);
```

Comment:
I guess it is the way "sed" is used to replace bulk values, I found myself that it needs at the end the "g" flag

sed 's/old/new/g'

otherwise it replaces only the first appearance of the work on each line.